dotcom-rendering/src/components/AffiliateDisclaimer.tsx

```diff
@@ -44,10 +44,8 @@ const ampStyles = css`
 
 const DisclaimerText = () => (
 	<p>
-		The Guardian’s product and service reviews are independent and are in no
-		way influenced by any advertiser or commercial initiative. We will earn
-		a commission from the retailer if you buy something through an affiliate
-		link.&nbsp;
+		The Guardian’s journalism is independent. We will earn a commission if
+		you buy something through an affiliate link.&nbsp;
```

Comment:
Could you elucidate the benefit of the non-breaking space here? Is it so there’s a space forced? Usually in JSX I’ve seen more frequent uses of `{' '}`…

Reply:
This wasn't my addition so I'm speculating - but I think the design showed that the _Learn more_ link shouldn't break onto it's own line - hence the use of a non breaking space